app/src/main/java/org/wildaid/ofish/ui/home/HomeFragment.kt

```diff
@@ -280,6 +280,7 @@ class HomeFragment : Fragment(R.layout.fragment_home),
         return when (event.dialogId) {
             ASK_CHANGE_DUTY_DIALOG_ID -> {
                 if (event.dialogBtn == DialogButton.POSITIVE) activityViewModel.onDutyChanged(true)
+                fragmentViewModel.boardVessel()
```

Comment:
This condition should be wrapped by bracers. Now User will be navigate to board Even if status Of duty